src/schema/paddle.ts

```diff
@@ -156,11 +156,11 @@ export const typeDefs = /* GraphQL */ `
     """
     Paddle platform ID
     """
-    paddle: String!
+    paddle: String
```

Comment:
We have to be careful now because now there can be a case where neither ID is defined, which is error prone.

Reply:
No worries, because if the ID is empty, then that is not going to be part of the request to the pricing preview endpoint. We will only request items when they are both present at the metadata and the response itself.